packages/quantic/force-app/main/default/lwc/quanticResultQuickview/quanticResultQuickview.js

```diff
@@ -37,6 +37,30 @@ export default class QuanticResultQuickview extends LightningElement {
   */
   @api maximumPreviewSize;
 
+  /**
+   * The icon to be shown in the preview button.
+   * @api
+   * @type {string}
+   * @defaultValue `'utility:preview'`
+   */
+  @api previewButtonIcon = 'utility:preview';
+
+  /**
+   * The label to be shown in the preview button.
+   * @api
+   * @type {string}
+   * @defaultValue `''`
+   */
+  @api previewButtonLabel = '';
+
+  /**
+   * The variant of the preview button.
+   * @api
+   * @type {''|'brand'|'outline'}
+   * @defaultValue `''`
+   */
+  @api previewButtonVariant = '';
```

Comment:
I think it would be better to default the variant and label props to undefined rather than empty strings. and add a conditional in the markup for rendering the label. `''` isn't really a variant and `''` doesn't accurately represent "no label" IMO.

Reply:
Variant could also be 'base' I guess too. Just an option.